src/collective/glossary/browser/static/main.js

```diff
@@ -1,6 +1,6 @@
 $(function(){
   $('#content-core').glossarizer({
-    sourceURL: '@@glossary',
+    sourceURL: portal_url + '/@@glossary',
```

Comment:
This javascript is a browser view, and the values of these variables are dynamic.

It will work for any Plone < 5 AFAIK